Quick/Configuration/QuickConfiguration.m

```diff
@@ -1,7 +1,10 @@
 #import "QuickConfiguration.h"
+#import "World.h"
 #import <Quick/Quick-Swift.h>
 #import <objc/runtime.h>
 
+QuickWorldBridge
```

Comment:
Is there a reason behind using a macro has opposed to just having `World.h` import the forward declarations?